samples/sensor/adt7420/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _adt7420:
+
+ADT7420: High accuracy digital I2C temperature sensor
+#####################################################
+
+Desciption
+**********
+
+This sample application periodically (1Hz) measures the ambient temperature
+in degrees Celsius. The result is written to the console.
+Optionally it also shows how to use the upper and lower threshold triggers.
+
+References
+**********
+
+ - ADT7420: http://www.analog.com/adt7420
+
+Wiring
+*******
+
+This sample uses the ADT7420 sensor controlled using the I2C interface.
+Connect Supply: **VDD**, **GND** and Interface: **SDA**, **SCL**
+optionally connect the **INT** to a interrupt capabale GPIO.
```

Comment:
add ``and optionally``

``capable``

Reply:
Will do. Thanks for the Review!
I already fixed and pushed them.